src/components/Questionnaire/Questions/Questions.vue

```diff
@@ -0,0 +1,95 @@
+<template>
+  <Card :class="$style.card">
+    <template #header>{{ questionnaire.title }}</template>
+  </Card>
+  <Card :class="$style.cardquestion">
+    <template #header>
+      <div v-for="(questioncontent, index) in questioncontents" :key="index">
+        <div>
+          <p>{{ questioncontent.body }}</p>
+          <span
+            ><IsRequired :required="questioncontent.is_required"></IsRequired
+          ></span>
+        </div>
+        <div v-if="questioncontent.question_type == 'Number'">
+          <ReadonlyNumber></ReadonlyNumber>
+        </div>
+        <div v-if="questioncontent.question_type == 'MultipleChoice'">
+          <ReadonlyRadio :contents="questioncontent.options"></ReadonlyRadio>
+        </div>
+        <div v-if="questioncontent.question_type == 'TextArea'">
+          <ReadonlyTextarea></ReadonlyTextarea>
+        </div>
```

Comment:
```suggestion
          <ReadonlyRadio v-if="questioncontent.question_type == 'MultipleChoice'" :contents="questioncontent.options"></ReadonlyRadio>
          <ReadonlyTextarea v-if="questioncontent.question_type == 'TextArea'"></ReadonlyTextarea>
```
こういう風にネストせずに`v-if`してほしい